Sources/PlayingCard.swift

```diff
@@ -31,13 +31,13 @@ public func ==(lhs: PlayingCard, rhs: PlayingCard) -> Bool {
 extension PlayingCard: Comparable {}
 
 public func <(lhs: PlayingCard, rhs: PlayingCard) -> Bool {
-    return lhs.suit < rhs.suit || (lhs.suit == rhs.suit && lhs.rank < rhs.rank)
+    return lhs.rank == rhs.rank ? lhs.suit == rhs.suit : lhs.rank < rhs.rank
```

Comment:
This isn't equivalent, this is no longer properly comparing cards when the rank is the same but the suit is different.